lib/v2/njust/maintainer.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    '/jwc/:type?': ['MilkShakeYoung', 'jasongzy'],
+    '/cwc/:type?': ['MilkShakeYoung', 'jasongzy'],
+    '/gs/:type?': ['MilkShakeYoung', 'jasongzy'],
+    '/eoe/:type?': ['jasongzy'],
+    '/dgxg/:type?': ['jasongzy'],
+    '/eo/:grade?/:type?': ['jasongzy'],
```

Comment:
```suggestion
    '/cwc/:type?': ['MilkShakeYoung', 'jasongzy'],
    '/dgxg/:type?': ['jasongzy'],
    '/eo/:grade?/:type?': ['jasongzy'],
    '/eoe/:type?': ['jasongzy'],
    '/gs/:type?': ['MilkShakeYoung', 'jasongzy'],
    '/jwc/:type?': ['MilkShakeYoung', 'jasongzy'],
```
Please sort it in alphabetical order